tests/test_shell.py

```diff
@@ -199,13 +324,234 @@ def test_remove(self):
+
+    def test_export(self):
+        self.call('python -m signac init my_project'.split())
+        project = signac.Project()
+        prefix_data = os.path.join(self.tmpdir.name, 'data')
+
+        err = self.call("python -m signac export {}"
+                        .format(prefix_data).split(), error=True)
+        assert 'No jobs to export' in err
+
+        for i in range(10):
+            project.open_job({'a': i}).init()
+        assert len(project) == 10
+        self.call("python -m signac export {}".format(prefix_data).split())
+        assert len(project) == 10
+        assert len(os.listdir(prefix_data)) == 1
+        assert len(os.listdir(os.path.join(prefix_data, 'a'))) == 10
+        for i in range(10):
+            assert os.path.isdir(os.path.join(prefix_data, 'a', str(i)))
+
+    def test_import(self):
+        self.call('python -m signac init my_project'.split())
+        project = signac.Project()
+        prefix_data = os.path.join(self.tmpdir.name, 'data')
+
+        err = self.call("python -m signac import {}"
+                        .format(self.tmpdir.name).split(), error=True)
+        assert 'Nothing to import.' in err
+
+        for i in range(10):
+            project.open_job({'a': i}).init()
+        job_ids = list(project.find_job_ids())
+        assert len(project) == 10
+        project.export_to(target=prefix_data, copytree=os.replace)
+        assert len(project) == 0
+        self.call("python -m signac import {}".format(prefix_data).split())
+        assert len(project) == 10
+        assert list(project.find_job_ids()) == job_ids
+
+        # invalid combination
+        with pytest.raises(ExitCodeError):
+            self.call('python -m signac import {} --sync-interactive --move'
+                      .format(prefix_data).split())
+
+    def test_import_sync(self):
+        project_b = signac.init_project('ProjectB', os.path.join(self.tmpdir.name, 'b'))
+        self.call('python -m signac init ProjectA'.split())
+        prefix_data = os.path.join(self.tmpdir.name, 'data')
+        project_a = signac.Project()
+        for i in range(4):
+            project_a.open_job({'a': i}).init()
+            project_b.open_job({'a': i}).init()
+        job_dst = project_a.open_job({'a': 0})
+        job_src = project_b.open_job({'a': 0})
+        job_src.document['a'] = 0
+        project_b.export_to(prefix_data)
+        err = self.call("python -m signac import {}".format(prefix_data).split(), error=True)
+        assert "Import failed" in err
+        self.call("python -m signac import {} --sync".format(prefix_data).split(), error=True)
+        assert len(project_a) == 4
+        assert 'a' in job_dst.document
+        assert job_dst.document['a'] == 0
+        out = self.call(
+            'python -m signac import {} --sync-interactive'.format(prefix_data),
+            'print(str(tmp_project), len(tmp_project)); exit()', shell=True)
+        assert 'ProjectA' in out
+        assert '4' in out
```

Comment:
Combine if possible.
```suggestion
        assert 'ProjectA 4' in out
```

Reply:
I don't we can combine both of them as `tmp_project.id` is combination of ` project.id, str(uuid.uuid4()) `

Reply:
Oh, okay. Nevermind. I'll resolve this.